src/Component/GeneralEntity/GeneralEntityRoot/GeneralEntityRoot.tsx

```diff
@@ -142,6 +148,11 @@ export function GeneralEntityRoot<T extends BaseEntity>({
   const [isFormLoading, setFormLoading] = useState<boolean>(false);
   const [isSaving, setIsSaving] = useState<boolean>(false);
   const [isUploadingFile, setIsUploadingFile] = useState<boolean>(false);
+  const [pageTotal, setPageTotal] = useState<number>();
+  const [pageSize, setPageSize] = useState<number>(10);
```

Comment:
we could make this configurable in `admin-client-config.js`

Reply:
or is this complicated because it may be linked to the styling, i.e. size of the table container?

Reply:
Good idea, will check that!

Reply:
See cd1c80e.